.devcontainer/dockerBuild.sh

```diff
@@ -0,0 +1,14 @@
+#environment setup
+sudo chmod -R a+rwx /etc/bash.bashrc
+echo "export PATH=\"/home/node/.cache/pypoetry/virtualenvs/gbstats-vabhrsvx-py3.7/bin:$PATH:$HOME/.poetry/bin\"" >> /etc/bash.bashrc
+
+#poetry installation
+curl -sSL https://raw.githubusercontent.com/python-poetry/poetry/master/get-poetry.py | python3 -
```

Comment:
This seems like it should be in the Dockerfile and not in the start script

Reply:
Moving this to the Dockerfile makes poetry install in the /root directory which causes a bunch of extra code around changing permissions.